proto/micro_rpc/options.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package oak.micro_rpc;
+
+import "google/protobuf/descriptor.proto";
+
+// Custom option extension related to micro_rpc
+extend google.protobuf.MethodOptions {
+  // Service methods that will be used with micro_rpc should provide a
+  // unique-per-service method_id number using this custom option.
+  optional int32 urpc_method_id = 425000010;
```

Comment:
didn't we settle on just `method_id` for this?

Reply:
Yep, thanks. I had this commit sitting around before I made the change in Google3.